molgenis-python/src/main/resources/templates/molgenis.py.ftl

```diff
@@ -158,3 +175,18 @@ class Session():
         z = x.copy()
         z.update(y)
         return z
+
+    def _raise_for_status(self, response):
+        error = None
+        try:
+            if 'errors' in response.json() and 'message' in response.json()['errors'][0]:
+                error = (response.json()['errors'][0]['message'])
+        except json.decoder.JSONDecodeError:
+            pass
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            if error:
+                raise requests.exceptions.HTTPError(str(e)+'\n'+error)
```

Comment:
When I add a duplicate value this is the traceback I get from raise_for_status:

Traceback (most recent call last):
  File "/Users/NPK/UMCG/git_projects/parse_rnaseq_output/run_parser.py", line 188, in <module>
    parse_output.parse_hisat(rundir_QC,connection,package=package)
  File "/Users/NPK/UMCG/git_projects/parse_rnaseq_output/PublicRNAseqParser/parse_output.py", line 291, in parse_hisat
    added_id = connection.add(package+'Hisat', data)[0]
  File "/Users/NPK/UMCG/git_projects/parse_rnaseq_output/PublicRNAseqParser/molgenis_wrapper.py", line 176, in add
    added_ids = self.session.add_all(entity_name, sanitized_data_list)
  File "/Users/NPK/UMCG/git_projects/parse_rnaseq_output/PublicRNAseqParser/molgenis.py", line 123, in add_all
    response.raise_for_status();
  File "/usr/local/lib/python3.5/site-packages/requests/models.py", line 837, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://localhost:8080/api/v2/public_rnaseq_4_Hisat

This does not include why there was a bad request. response.json() has this:

{'errors': [{'message': "Duplicate value '1_internal_id_ATACseq-D_S3_L001' for unique attribute 'internalId_sampleid' from entity 'public_rnaseq_4_Hisat' (entity 1)"}]}